medusalib/src/main/java/com/trendyol/medusalib/navigator/MultipleStackNavigator.kt

```diff
@@ -158,6 +158,19 @@ open class MultipleStackNavigator(
         initializeStackState()
     }
 
+    override fun reset(tabIndex: Int, fragmentStackIndex: Int) {
+        val fragmentTagStack = fragmentStackState.fragmentTagStack[tabIndex]
+        val stackSize = fragmentTagStack.size - 1
+        if (stackSize > fragmentStackIndex) {
+            (stackSize downTo fragmentStackIndex + 1).forEach { position ->
+                fragmentManagerController.findFragmentByTagAndRemove(fragmentTagStack[position].fragmentTag)
+                fragmentTagStack.pop()
+            }
+            fragmentManagerController.commitAllowingStateLoss()
+            fragmentManagerController.enableFragment(getCurrentFragmentTag())
+        }
```

Comment:
How about else case? We can call reset(tabIndex: Int, resetRootFragment: Boolean = true) to reset the tab or throw an exception? Which one do you think would be appropriate for your usecase?